src/test/kotlin/com/igorwojda/integer/fizzbuzz/challenge.kt

```diff
@@ -4,7 +4,17 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun fizzBuzz(n: Int): List<String> {
-    TODO("not implemented")
+    val numberList = mutableListOf<String>()
+
+    (1..n).forEach() {
```

Comment:
Can you think of a way of creating the result list while iterating instead of appending to a mutable list?